dependencies.yaml

```diff
@@ -630,7 +630,7 @@ dependencies:
     common:
       - output_types: [conda, requirements, pyproject]
         packages:
-          - polars>=1.0
+          - polars>=1.3
```

Comment:
Polars 1.3 is the version that has the public collect UX, hence defines the minimum version we support.